src/main/groovy/com/deploygate/gradle/plugins/DeployGatePlugin.groovy

```diff
@@ -37,19 +43,37 @@ class DeployGatePlugin implements Plugin<Project> {
     void apply(Project project) {
         DeprecationLogger.reset()
 
-        setupExtension(project)
+        def credentialStore = new CliCredentialStore()
```

Comment:
To follow the dependency injection, we open the local credential store only once at this point. Plugin class will set these values to tasks if required.